x/slash/keeper.go

```diff
@@ -172,6 +172,7 @@ func (k Keeper) Slash(ctx sdk.Context, reason string, failedValidator staking.Va
 		}
 	}
 
+	beneficiaries = append(beneficiaries, NewAccountFractionPair("1", k.SyncerReward(ctx)))
```

Comment:
should not add compensation to guard slash, because the fallback guardian already has a fraction of reward from it.

Reply:
the beneficiary amount should be `max(slashAmoumt, slashSenderCompensation)`